anubis-management-api/src/policies/schemas.py

```diff
@@ -30,9 +30,11 @@ class PolicyBase(BaseModel):
     # agent iri (unless pre-defined) are agent_type_iri:id
 
     @validator('agent')
-    def valid_agent(cls, agent):
+    def valid_agent(cls, agent, values):
         if agent is None or agent == []:
             raise ValueError('you need to pass at least one agent')
+        if "access_to" in values and values["access_to"] == "default" and (len(agent) != 1 or agent[0] != "foaf:Agent"):
+            raise ValueError('when access_to is set to default, only agent permitted is foaf:Agent')
```

Comment:
i think we could have default for any agent, not only foaf how is this in WAC?

Reply:
ok, so basically, point being is that, the serialization to WAC is what needs to be changed, rather than the internal API correct?

-> any policy that have a * in entity and entity-type is a a default policy.

still, for the "container" discussion, do we support `/servicePath/*` ?